ivy/functional/ivy/creation.py

```diff
@@ -1076,14 +1076,24 @@ def linspace(
     >>> y = ivy.array([4,5])
 
     >>> ivy.linspace(x, y, 4)
-    ivy.array([[1, 2, 3, 4],[2, 3, 4, 5]])
+    ivy.array([[1, 2, 3, 4], [2, 3, 4, 5]])
 
     >>> ivy.linspace(x, y, 4, axis = 0)
     ivy.array([[1, 2],
                [2, 3],
                [3, 4],
                [4, 5]])
 
+    Using array instance method
```

Comment:
Could you add/change just the following line(s):
```
    Instance Method Examples
    -------------------
    With :class:`ivy.Array` input:
```

Reply:
Done